composer.json

```diff
@@ -22,7 +22,8 @@
     "zendframework/zend-diactoros": "~1.1",
```

Comment:
me too.

I noticed a question about apigilities minimum requirement? think you want an answer first?